### ISSUE TITLE
NomadProblem not freed in C interface

### ISSUE THREAD

**Issue by paulapatience** [bbopt/nomad#160]
A NomadProblem is allocated in [createNomadProblem](https://github.com/bbopt/nomad/blob/4cfa402d8a7dcfa8a8b1ad4b98111d88242d8fe4/interfaces/CInterface/NomadStdCInterface.cpp#L96) but does not seem to be freed in [freeNomadProblem](https://github.com/bbopt/nomad/blob/4cfa402d8a7dcfa8a8b1ad4b98111d88242d8fe4/interfaces/CInterface/NomadStdCInterface.cpp#L111). Am I missing something?

**Comment by salomonl** [comment]
Not really. It is a mistake of our part. We will fix it. Thank you for spotting it.

**Comment by salomonl** [comment]
Corrected for the next release.
